### DISCUSSION TITLE
Android:Google Input 空格键无法调节方向 & 编辑框中无法局部选择

### DISCUSSION THREAD

**Discussion by Darleey** [tiddly-gittly/TidGi-Desktop#92] [Q&A] [answered]
## Environment

Version v0.4.1

Electron Version: 14.0.0-beta.12
Node Version: 14.17.0
Chromium Version: 93.0.4557.4

OS (macOS, Windows, Linux) and Version:

local server: Windows 10 Desk
client: Android 11 Chrome Browser

## Description
在 Android 设备的浏览器中，无法通过 谷歌键盘 的空格键[左右滑动]来调整光标的位置。在官方提供的 tiddlywiki 中，这种操作是可行的～

**Comment by linonetwo** [comment, marked as answer]
感觉可能是 codemirror 插件导致的吧，官方wiki中这个插件装了但是没有启用，你可以启用一下对照看看。

这个应该基本都是插件导致的问题。当然，我这里面装的wiki是最新预发布版，也有可能是最新的 TiddlyWiki 的bug。

**Reply by Darleey** [reply]
厉害，确实是 codemirror 的锅！

**Reply by linonetwo** [reply]
这个想修就得去官方仓库或者codemirror仓库提issue了。
不过我移动端一般是语音输入，然后电脑端有空再改细节。
